src/components/Items/Question/Footer.vue

```diff
@@ -0,0 +1,192 @@
+  components: { IconButton },
+  props: {
+    isAnswerSubmitted: {
+      // whether the answer has been submitted
+      default: false,
+      type: Boolean,
+    },
+    isAnswerCorrect: {
+      // if the answer has been submitted, is it correct
+      default: false,
+      type: Boolean,
+    },
+    isOptionSelected: {
+      // whether an option has been selected
+      default: false,
+      type: Boolean,
+    },
+    isFullscreen: {
+      // whether the original modal is in fullscreen
+      default: false,
+      type: Boolean,
+    },
+  },
+  computed: {
+    localIsFullscreen: {
+      // local copy of isFullscreen prop
+      get() {
+        return this.isFullscreen;
+      },
+      set(localIsFullscreen) {
+        this.$emit("update:isFullscreen", localIsFullscreen);
+      },
+    },
+    submitButtonTitleConfig() {
+      // config for the text of the submit button
+      return {
+        value: "✓ " + this.$t("player.question.submit"),
+        class: "text-white text-lg sm:text-xl lg:text-2xl font-bold",
+      };
+    },
+    submitButtonClass() {
+      // class for the submit button
+      return "bg-green-500 hover:bg-green-600 ring-green-500 p-1 pl-4 pr-4 sm:p-2 sm:pl-6 sm:pr-6 lg:p-4 lg:pl-10 lg:pr-10 rounded-md shadow-xl disabled:opacity-50 disabled:pointer-events-none";
+    },
+    reviseButtonTitleConfig() {
+      // config for the text of the revise button
+      return {
+        value: "⟳ " + this.$t("player.question.revise"),
+        class: "text-blue-500 text-lg sm:text-xl lg:text-2xl font-bold",
+      };
+    },
+    reviseButtonClass() {
+      // class for the revise button
+      return "hover:bg-gray-200 ring-blue-500 p-1 pl-4 pr-4 sm:p-2 sm:pl-6 sm:pr-6 lg:p-4 lg:pl-10 lg:pr-10 rounded-md shadow-xl";
+    },
+    proceedButtonTitleConfig() {
+      // config for the text of the proceed button
+      return {
+        value: "⟳ " + this.$t("player.question.proceed"),
+        class: "text-green-500 text-lg sm:text-xl lg:text-2xl font-bold",
+      };
+    },
+    proceedButtonClass() {
+      // class for the proceed button
+      return "hover:bg-gray-200 ring-green-500 p-1 pl-4 pr-4 sm:p-2 sm:pl-6 sm:pr-6 lg:p-4 lg:pl-10 lg:pr-10 rounded-md shadow-xl";
+    },
+    fullscreenIconConfig() {
+      // icon config for the fullscreen button
+      return {
+        enabled: true,
+        iconName: this.fullscreenIconName,
+        iconClass: "h-5 w-5 sm:h-8 sm:w-8 md:h-10 md:w-10 shadow-none hover:bg-gray-200",
+      };
+    },
+    fullscreenButtonClass() {
+      // class for the fullscreen button
+      return "ring-transparent";
+    },
+    fullscreenIconName() {
+      // icon to show if the answer was correct or not
+      if (this.isFullscreen) {
+        return "exit-fullscreen";
+      }
+      return "enter-fullscreen";
+    },
+    answerCorrectnessIcon() {
+      // icon to show if the answer was correct or not
+      var icon = require("@/assets/images/window-close-solid.svg");
```

Comment:
```suggestion
      var icon = require("@/assets/images/times-solid.svg");
```

We were using this before